doc/osx.t2t

```diff
@@ -90,7 +90,7 @@ Alternatively you can download and install Qt Open Source for MacOS from the
 
 https://www.qt.io/
 
-in the same version as referenced in the install script.
+in the same version as referenced in the install script. It must be installed in /opt/Qt
```

Comment:
I do not see this change in the INSTALL ... I think easiest approach for such small change it to manually add it to both INSTALL and osx.t2t file (and revert all other changes in INSTALL file)